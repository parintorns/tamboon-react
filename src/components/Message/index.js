import React from 'react'

import { Wrapper } from './styles'

export default function Message({ message }) {
  return <Wrapper>{message}</Wrapper>
}
