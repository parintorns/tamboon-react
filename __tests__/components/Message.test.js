import React from 'react'
import { shallow } from 'enzyme'

import Message from '../../src/components/Message'

let wrapper

const props = {
  message: 'This is a message'
}

beforeEach(() => {
  wrapper = shallow(<Message {...props} />)
})

describe('Render', () => {
  it('should render Spinner', () => {
    expect(wrapper.debug()).toMatchSnapshot()
  })
})
