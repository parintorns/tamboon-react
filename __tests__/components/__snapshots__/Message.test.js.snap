// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render should render Spinner 1`] = `
"<styled.div>
  This is a message
</styled.div>"
`;
